content/_components/rest-api-component.md

```diff
@@ -177,23 +203,24 @@ Rest-api component automatically load binary data to attachments with next conte
 * application/octet-stream
 
 ## Exception handling
-Rest API component uses exception handling logic below:
+Rest API component uses exception handling logic below: 
 ![Exception handling logic](https://user-images.githubusercontent.com/13310949/41960520-9bd468ca-79f8-11e8-83f4-d9b2096deb6d.png)
 
 ## Known Limitations
 
-The component can parse any of json and xml content types.
+The component can parse any of json and xml content types. 
 There are:
 * application/json
 * application/xml
 * text/xml
 * etc.
 
-`If content type is not  exists  in response header, component will try parse response as json.
+`If content type is not  exists  in response header, component will try parse response as json. 
 If it get parse exception, it return response as is.`
 
 > Make sure not to perform your tests using the [requestb.in](https://requestb.in/) since it responds with the `content-type: text/html`.
 
-Here are some further limitation of the REST API component:
+Attachments limitations:
 
-*   The component can't handle multi-part responses
+1. Maximal possible size for an attachment is 10 MB.
+2. Attachments mechanism does not work with [Local Agent Installation](https://support.{{site.data.tenant.name}}/support/solutions/articles/14000076461-announcing-the-local-agent-)
```

Comment:
Please remove this link since this article is very outdated and is not reflecting the real description of the subject. Just write: `Attachments mechanism does not work with the Local Agents`